node/typings/index.d.ts

```diff
@@ -1,4 +1,3 @@
-/// <reference path="globals/jsonschema/index.d.ts" />
 /// <reference path="globals/mz/index.d.ts" />
 /// <reference path="globals/node/index.d.ts" />
 /// <reference path="globals/xml2js/index.d.ts" />
```

Comment:
Does this change resolve the typings issue I faced, when test script was in typescript ?